menas/ui/service/RunService.js

```diff
@@ -91,6 +91,7 @@ var RunService = new function () {
     oRunSummaries.forEach(run => {
       run.status = this._normalizeStatus(run.status)
     });
+    oRunSummaries.sort((a, b) => b.runId - a.runId);
```

Comment:
please sort by the start time. Now it looks like this and it is still a bit hard to understand. Even more, since each version has it's own id's. So something run for the first time would end on the end.

![image](https://user-images.githubusercontent.com/17467768/54769314-405c9b80-4c01-11e9-83ad-c4a7148c83ac.png)